pyproject.toml

```
@@ -75,8 +75,8 @@ experimental = [
   "tenacity",
 ]
 llama-index = [
-  "llama-index==0.9.45",  # always pin to a version that keeps our notebooks working
-  "openinference-instrumentation-llama-index==0.1.3",
+  "llama-index==0.10.3",  # always pin to a version that keeps our notebooks working
```

```suggestion
  "llama-index~=0.10.3",  # always pin to a version that keeps our notebooks working
```

Probably could pin a little more loosely at this point.